presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -102,7 +103,7 @@ public HashAggregationOperatorFactory(
                     new SpillerFactory()
                     {
                         @Override
-                        public Spiller create(List<Type> types)
+                        public Spiller create(List<Type> types, SpillContext spillContext, AggregatedMemoryContext memoryContext)
```

Comment:
There is a subclass of `SpillerFactory` in this class that contains an unused method `getTotalSpilledBytes`.